gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/factories/GafferFactory.java

```diff
@@ -101,89 +94,34 @@ private static GafferSpec overrideGafferSpecConfig(final GafferSpec config, fina
-        opAuthoriser.put("class", OperationAuthoriser.class.getName());
-        opAuthoriser.put("auths", formattedAuths);
+    private static OperationDeclarations createOperationDeclarations(final GafferSpec federatedSpec) {
+        OperationDeclarations existingOperationDeclarations = null;
 
-        return opAuthoriser;
-    }
+        final OperationDeclarations declarations = new OperationDeclarations.Builder()
+                .declaration(new OperationDeclaration.Builder()
+                        .handler(new GetProxyUrlHandler())
+                        .operation(GetProxyUrl.class)
+                        .build())
+                .build();
 
-    private static Map<String, List> getFormattedAuths(final Object existingAuths) {
-        final Map<String, List> formattedAuths = new LinkedHashMap<>();
-        Map<String, Object> notFormattedAuths = new LinkedHashMap<>();
-        if (existingAuths != null) {
-            final List<Object> configResult = (ArrayList) existingAuths;
-            for (final Object key : configResult) {
-                if (((LinkedHashMap) key).get("class") != null && ((LinkedHashMap) key).get("class").equals(OperationAuthoriser.class.getName())) {
-                    notFormattedAuths = (LinkedHashMap) ((LinkedHashMap) key).get("auths");
-                }
-            }
-            if (notFormattedAuths != null) {
-                notFormattedAuths.forEach((key, value) -> {
-                    List<String> auths = formatExistingAuths(key, value);
-                    if (key != null && auths != null && !auths.isEmpty()) {
-                        formattedAuths.put(key, auths);
-                    }
-                });
-            }
+        if (federatedSpec != null && federatedSpec.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY) != null) {
+            existingOperationDeclarations = (OperationDeclarations) federatedSpec.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY);
         }
-        return formattedAuths;
-    }
 
-    private static Set<Object> createOperationDeclaration(final GafferSpec config) {
-        final List<Object> operations = new ArrayList<>();
-        if (config.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY) != null) {
-            operations.add(config.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY));
+        if (existingOperationDeclarations != null) {
+            List<OperationDeclaration> operations = existingOperationDeclarations.getOperations();
+            operations.forEach((operation) -> declarations.getOperations().add(operation));
         }
-        final Map<String, Object> proxyUrlDeclaration = new HashMap<>();
-        final Map<String, String> proxyUrlClass = new HashMap<>();
-
-        proxyUrlClass.put("class", "uk.gov.gchq.gaffer.proxystore.operation.handler.GetProxyUrlHandler");
 
-        proxyUrlDeclaration.put("operation", "uk.gov.gchq.gaffer.proxystore.operation.GetProxyUrl");
```

Comment:
Why is GetProxyUrl being removed?

Reply:
Replaced with this code line: 108-113

  final OperationDeclarations declarations = new OperationDeclarations.Builder()
                .declaration(new OperationDeclaration.Builder()
                        .handler(new GetProxyUrlHandler())
                        .operation(GetProxyUrl.class)
                        .build())
                .build();